Readme.md

```diff
@@ -326,6 +326,8 @@ The `deviceId` is now exposed through the context object in each request and can
 * `this.emit(':tell', 'Sometimes when I look at the Alexa skills you have all taught me, I just have to say, <say-as interpret-as="interjection">Bazinga.</say-as> ');`
 * `this.emit(':tell', '<say-as interpret-as="interjection">Oh boy</say-as><break time="1s"/> this is just an example.');`
 
+_Keep in mind that speechcons can be used only in English (en-US) skills. They are not currently available for English (en-UK) or German (DE) skills._
```

Comment:
Small edit: change to English (US), English (UK) and German to align with developer console and [speechcon documentation](https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/speechcon-reference)